### DISCUSSION TITLE
Add pagination and side-scrollbar

### DISCUSSION THREAD

**Discussion by Kelvin-Data** [Akascape/CTkTable#80] [Ideas]
I have 35 students record to display, however, the table details become very small for viewing. I am suggested to add a pagination and side-scrollbar to limit the  number of row for viewing.

**Comment by Akascape** [comment]
@Kelvin-Data Pagination can be added manually by updating the table data.
And we can add the table in a scrollable frame of customtkinter to limit the number of row for viewing.
